From 1c83b7de944f259087f7a5cca7c24e5b50938015 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Wed, 27 May 2020 14:09:42 +0800
Subject: [PATCH] ARM: configs: rv1126_defconfig: set CONFIG_CMA_SIZE_MBYTES=8

For RV1126 evb, total dma alloc is 1886 pages, about ~8MiB, so
reserved 8MiB for linux default cma pool.

Change-Id: I33c1d054c3862facb1eefc898aa79491864c4ae5
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm/configs/rv1126_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/rv1126_defconfig b/arch/arm/configs/rv1126_defconfig
index d09250c30112..69abf4ad1f12 100644
--- a/arch/arm/configs/rv1126_defconfig
+++ b/arch/arm/configs/rv1126_defconfig
@@ -86,7 +86,7 @@ CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 # CONFIG_ALLOW_DEV_COREDUMP is not set
 CONFIG_DMA_CMA=y
-CONFIG_CMA_SIZE_MBYTES=1
+CONFIG_CMA_SIZE_MBYTES=8
 CONFIG_CMA_ALIGNMENT=4
 CONFIG_MTD=y
 CONFIG_MTD_CMDLINE_PARTS=y
-- 
2.35.3

